Presentation.Web/app/components/it-contract/it-contract-plan.controller.ts

```diff
@@ -240,16 +240,25 @@
                                     "AssociatedSystemRelations($select=Id)," +              //Only using the length, so select 1 field
                                     "Reference($select=URL,Title,ExternalReferenceId)," +
                                     "LastChangedByUser($select=Name,LastName)";
+
                                 // if orgunit is set then the org unit filter is active
                                 var orgUnitId = this.$window.sessionStorage.getItem(this.orgUnitStorageKey);
+                                var filterParameters = "";
+                                var query;
+                                // if orgunit is set then the org unit filter is active
                                 if (orgUnitId === null) {
-                                    return `/odata/Organizations(${this.user.currentOrganizationId})/ItContracts` +
-                                        urlParameters;
+                                    query = `/odata/Organizations(${this.user.currentOrganizationId})/ItContracts`;
                                 } else {
-                                    return `/odata/Organizations(${this.user.currentOrganizationId
-                                        })/OrganizationUnits(${orgUnitId})/ItContracts` +
-                                        urlParameters;
+                                    query = `/odata/Organizations(${this.user
+                                        .currentOrganizationId})/OrganizationUnits(${orgUnitId})/ItContracts`;
                                 }
+
+                                var procurementId = this.$window.sessionStorage.getItem(this.procurementInitiatedStorageKey);
+                                if (procurementId !== null) {
+                                    filterParameters += `&%24filter=ProcurementInitiated eq '${procurementId}'`;
```

Comment:
Look at how the "undecided/blank" option is handled in DPR .. it must match both null case and "blank" case